frontend/src/app/contribute.cljs

```diff
@@ -58,40 +77,38 @@
                    (do
                      (reset! error-title (:error data))
                      (reset! error-description (:description data)))
-                   (do
-                     (reset! backend-data data)
-                     (reset! log (:content (:log data)))
-                     (reset! build-id (:build_id data))
-                     (reset! build-id-title (:build_id_title data))
-                     (reset! build-url (:build_url data))
-                     (reset! spec (:spec_file data))
-                     (reset! container (:container_file data))
-
-                     (reset!
-                      files
-                      (vec (map (fn [log]
-                                  ;; We must html encode all HTML characters
-                                  ;; because we are going to render the log
-                                  ;; files dangerously
-                                  (update log :content #(.encode html-entities %)))
-                                (:logs data))))
-
-                     (reset! error-title nil)
-                     (reset! error-description nil))))))))
+                   (set-atoms data)))))))
+
+(defn fetch-logs-upload []
+  (let [data {:build_id_title "Upload"
+              :logs [{:name (.getItem js/localStorage "name")
+                      :content (.getItem js/localStorage "content")}]
+              :spec_file {:name "fake_spec_name.spec"
```

Comment:
can we return here `nil` instead of fake spec file? to avoid the same problem as #53

Reply:
Nice, done :-)